File: review-engine/examples/rule_self_tests/cases/go/shared_security/violating.go
package security

import (
	"fmt"
	"os/exec"
)

var api_key = "sk_live_go_fixture"

func loadUser(db DB, userID string, table string) error {
	command := "id " + userID
	_ = exec.Command("sh", "-c", command).Run()
	query := fmt.Sprintf("SELECT * FROM %s WHERE id = %s", table, userID)
	return db.Query(query)
}

File: review-engine/examples/rule_self_tests/cases/python/shared_security/violating.py
import subprocess


def load_user(conn, user_id, table):
    api_key = "sk_live_python_fixture"
    command = "id " + user_id
    subprocess.run(command, shell=True)
    query = f"SELECT * FROM {table} WHERE id = %s"
    return conn.execute(query, (user_id,))
